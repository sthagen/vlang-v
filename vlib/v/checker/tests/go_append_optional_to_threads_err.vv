fn foo() ! {}

fn main() {
	mut ths := []thread{}
	ths << go foo()
	ths.wait()
}
