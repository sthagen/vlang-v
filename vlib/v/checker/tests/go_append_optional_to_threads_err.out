vlib/v/checker/tests/go_append_optional_to_threads_err.vv:5:12: error: cannot append `thread !` to `[]thread`
    3 | fn main() {
    4 |     mut ths := []thread{}
    5 |     ths << go foo()
      |               ~~~~~
    6 |     ths.wait()
    7 | }
